sql/_13_issues/_20_1h/cases/cbrd_23689.sql

```diff
@@ -0,0 +1,25 @@
+set system parameters 'dont_reuse_heap_file=yes';
+drop table if exists tmp1;
+drop table if exists tmp2;
+
+CREATE TABLE tmp1 (col1 VARCHAR(10), col2 VARCHAR(10));
+CREATE TABLE tmp2 (col1 VARCHAR(10), col2 VARCHAR(10));
+INSERT INTO tmp1 SELECT ROWNUM, (ROWNUM +1)/10000 FROM db_class a, db_class b, db_class c LIMIT 10000;
+INSERT INTO tmp2 SELECT ROWNUM, ROWNUM FROM db_class a, db_class b, db_class c LIMIT 10000;
```

Comment:
When using the db_class table, fewer rows can be entered. It is more accurate to do the following:
```
from table({0,1,2,3,4,5,6,7,8,9}) a, table({0,1,2,3,4,5,6,7,8,9}) b, table({0,1,2,3,4,5,6,7,8,9}) c, table({0,1,2,3,4,5,6,7,8,9}) d
```